app/models/category.rb

```diff
@@ -0,0 +1,5 @@
+class Category < ApplicationRecord
+  validates :title, presence: true
+
+  enum kind: %i[regular irregular]
```

Comment:
remove this column, because we can define a category type based on user_id column. example:  if category has user_id this means that category custom and belongs to user.